scripts/dev/e2e.py

```diff
@@ -179,47 +217,12 @@ def wait_for_pod_to_be_running(
         lambda: corev1.read_namespaced_pod(name, namespace),
         lambda pod: pod.status.phase == "Running",
         sleep_time=5,
-        timeout=90,
+        timeout=180,
```

Comment:
Why did you increase the timeout here?

Reply:
The e2e-test pod takes longer to get started than the testrunner did!